poetry/spdx/__init__.py

```diff
@@ -6,33 +6,40 @@
 from .license import License
 from .updater import Updater
 
-_licenses = None
+_licensesByShortIdentifier = None
+_licensesByLongName = None
```

Comment:
Why separate them into two variables? why not store both the long and the short license names in the same dict?

Reply:
Because it's easier to look them up that way.